jina/peapods/pods/k8slib/kubernetes_tools.py

```diff
@@ -89,13 +89,15 @@ def networking_v1_beta1_api(self):
 
 def create(
     template: str,
```

Comment:
I think the parameters `kind` and template are redundant. Let's only keep one of them.